### ISSUE TITLE
集群怎么设置？不成功？

### ISSUE THREAD

**Issue by w3yyb** [emqtt/emqttd#108]
设置'etc/vm.args':

192.168.6.156上
-name emqttd@192.168.6.156
./bin/emqtt start    启动不成功。
-name emqttd@127.0.0.1 
./bin/emqtt start    启动成功。

2台不能都设为 -name emqttd@127.0.0.1   这个集群就设置不成功了。
如何设置，谢谢

**Comment by w3yyb** [comment]
if I set -name emqttd@host1  can't start the emqttd
print:
emqttd failed to start within 15 seconds,
see the output of 'emqttd console' for more information.
If you want to wait longer, set the environment variable
WAIT_FOR_ERLANG to the number of seconds to wait.

**Comment by erylee** [comment]
please rm data/mnesia/* first if you changed node name.

**Comment by w3yyb** [comment]
thanks

**Comment by w3yyb** [comment]
我试了，仍然不能成功。

**Comment by erylee** [comment]
cannot start still?

**Comment by w3yyb** [comment]
yes ,can't start     -name emqttd@192.168.6.156  or -name emqttd@host1

**Comment by erylee** [comment]
first, my etc/vm.args config:
```
-name emqttd@192.168.18.124
```

second:

```
rm -rf data/mnesia/*
```

and then:

./bin/emqttd console

```
airlee:emqttd erylee$ ./bin/emqttd console
Exec: /Users/erylee/Projects/emqttd/rel/emqttd/erts-6.3/bin/erlexec -boot /Users/erylee/Projects/emqttd/rel/emqttd/releases/0.6.0/emqttd             -embedded -config /Users/erylee/Projects/emqttd/rel/emqttd/etc/app.config             -pa ./lib/basho-patches             -args_file /Users/erylee/Projects/emqttd/rel/emqttd/etc/vm.args -- console
Root: /Users/erylee/Projects/emqttd/rel/emqttd
Erlang/OTP 17 [erts-6.3] [source] [64-bit] [smp:4:4] [async-threads:32] [hipe] [kernel-poll:true]


=INFO REPORT==== 24-Apr-2015::15:36:04 ===
    alarm_handler: {set,{system_memory_high_watermark,[]}}
starting emqttd on node 'emqttd@192.168.18.124'
emqttd config is starting...[done]
emqttd event is starting...[done]
emqttd pooler is starting...[done]
emqttd client manager is starting...[done]
emqttd session manager is starting...[done]
emqttd session supervisor is starting...[done]
emqttd pubsub is starting...[done]
emqttd broker is starting...[done]
emqttd metrics is starting...[done]
emqttd bridge supervisor is starting...[done]
emqttd access control is starting...[done]
emqttd system monitor is starting...[done]
15:36:05.245 [info] mqtt listen on 0.0.0.0:1883 with 16 acceptors.
15:36:05.246 [info] mqtts listen on 0.0.0.0:8883 with 4 acceptors.
15:36:05.256 [info] http listen on 0.0.0.0:8083 with 4 acceptors.
Erlang MQTT Broker 0.6.2 is running now
Eshell V6.3  (abort with ^G)
(emqttd@192.168.18.124)1> node().
'emqttd@192.168.18.124'
```

**Comment by w3yyb** [comment]
[root@cndkbtamqpusha02 emqttd]# ./bin/emqttd console
Exec: /opt/emqttd/erts-6.3/bin/erlexec -boot /opt/emqttd/releases/0.6.0/emqttd             -embedded -config /opt/emqttd/etc/app.config             -pa ./lib/basho-patches             -args_file /opt/emqttd/etc/vm.args -- console
Root: /opt/emqttd
Erlang/OTP 17 [erts-6.3] [source] [64-bit] [smp:4:4] [async-threads:64] [hipe] [kernel-poll:true]

starting emqttd on node 'emqttd@192.168.6.157'
emqttd config is starting...[done]
emqttd event is starting...[done]
emqttd pooler is starting...[done]
emqttd client manager is starting...[done]
emqttd session manager is starting...[done]
emqttd session supervisor is starting...[done]
emqttd pubsub is starting...[done]
emqttd broker is starting...[done]
emqttd metrics is starting...[done]
emqttd bridge supervisor is starting...[done]
emqttd access control is starting...[done]
emqttd system monitor is starting...[done]
16:10:46.349 [info] mqtt listen on 0.0.0.0:1885 with 64 acceptors.
16:10:46.350 [info] mqtts listen on 0.0.0.0:8883 with 4 acceptors.
16:10:46.357 [info] http listen on 0.0.0.0:8083 with 4 acceptors.
Erlang MQTT Broker 0.6.1 is running now
Eshell V6.3  (abort with ^G)
(emqttd@192.168.6.157)1> 16:11:26.284 [info] metrics PUBLISH to $SYS/brokers/emqttd@192.168.6.157/bytes/received
16:11:26.284 [info] metrics PUBLISH to $SYS/brokers/emqttd@192.168.6.157/bytes/sent

**Comment by w3yyb** [comment]
OK, now  emqttd@192.168.6.157 successful!!!!!!!!!

**Comment by erylee** [comment]
Hi, Yang, I released 0.6.2, thanks for your bug report.
